File: MyLeetCode/Leetcode/Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        char last_char = ' ';
        int last_len = 0;
        for (int i = 0; i < s.size(); i++)
        {
            if (s[i] != ' ')
            {
                if (last_char == ' ')
                    last_len = 1;
                else
                    last_len++;
            }
            last_char = s[i];
        }
        return last_len;
    }
};

File: MyLeetCode/Leetcode/Length of Last Word.py
class Solution:
    # @param s, a string
    # @return an integer
    def lengthOfLastWord(self, s):
        if (s == ''):
            return 0;
        word = s.split();
        if (len(word) == 0):
            return 0;
        else:
            return len(word[-1]);
